Mathlib/Algebra/Order/Floor.lean

```diff
@@ -499,6 +499,21 @@ theorem floor_div_eq_div (m n : ℕ) : ⌊(m : α) / n⌋₊ = m / n := by
 
 end LinearOrderedSemifield
 
+section LinearOrderedField
+variable [LinearOrderedField α] [FloorSemiring α] {a b : α}
+
+lemma ceil_lt_mul (ha : 0 ≤ a) (hb : 1 < b) (h : (b - 1)⁻¹ ≤ a) : ⌈a⌉₊ < b * a := by
```

Comment:
We should have `le` versions of these too

Reply:
What would the `le` version be? I'm pretty sure it would have exactly the same assumptions

Reply:
I needed something stronger at some point but I can't find it right now - let's leave it for now